recipes/imageio/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.1.1" %}
+
+package:
+  name: imageio
+  version: {{ version }}
+
+source:
+  fn: imageio-{{ version }}.zip
+  url: https://pypi.python.org/packages/71/67/a6a3192355e9ffabb16e84695592d51fdde6d4d884a219509bea8560876e/imageio-2.1.1.zip
```

Comment:
This hash-based url is version specific and has the version hard-coded in it. Please update to follow the [example]( https://github.com/conda-forge/staged-recipes/blob/4dcfd3d829690585881e3155314b62a73100c30a/recipes/example/meta.yaml#L18 ).